### PR TITLE
Added a login page and component

### PR DESCRIPTION
**Pull request by valdemarbang**

### CODE REVIEW

File: src/components/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    // Get form data, if not available set to empty string
+    const email = (data.get("email") || "") as string;
+    const password = (data.get("password") || "") as string;
+
+    // Reset all errors before validation
+    setEmailError("");
+    setPasswordError("");
+
+    // Simple form validation
+    let hasErrors = false; // shitty way of doing this, but it works
```
Comment (Jacalz) [inline src/components/Login.tsx:39]: How about just returning each time you do `hasErrors = true` instead?

File: src/components/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    if (!response.ok) {
+      if (responseData.emailError) {
+        setEmailError(responseData.emailError);
+      }
+      if (responseData.passwordError) {
+        setPasswordError(responseData.passwordError);
+      }
+    }
+    // sucessful login
+    else {
```
Comment (Jacalz) [inline src/components/Login.tsx:75]:
```suggestion
    } else { // sucessful login
```

**Review by Jacalz — CHANGES_REQUESTED** [review]
Wow. Nice work. I just left two minor suggestions inline :)

File: src/components/Login.tsx (hunk 1 above)
Comment (Jacalz) [reply src/components/Login.tsx:39]: Good point. Thanks for explaining

**Review by Jacalz — DISMISSED** [review]

**Review by emilloof — APPROVED** [review]

**Review by Jacalz — APPROVED** [review]

**Review by Jacalz — APPROVED** [review]

**Review by emilloof — APPROVED** [review]

**Pull request merged**